submit/main/fund.py

```diff
@@ -108,11 +108,11 @@ def get_active_funds(self, roles: list[str]):
 PATHFINDERS_APP_CONFIG = FundConfig(
     fund_name="Pathfinders",
     user_role="PF_MONITORING_RETURN_SUBMITTER",
-    current_reporting_period="January to March 2024",
-    current_reporting_round=1,
-    current_deadline=datetime.date(day=30, month=4, year=2024),
+    current_reporting_period="April to September 2024",
+    current_reporting_round=2,
+    current_deadline=datetime.date(day=1, month=11, year=2024),
```

Comment:
I'd like to chat this block through with you. It feels like this should be fed from ReportingRound - but it gets a bit complex on what logic we should use to determine which rounds are 'open'.

In my head it should be 'any rounds where the current date+time is between submission_period_start and submission_period_end'. But we should talk through how this would feed through everywhere. It feels "right" to me though.

Probably can be done in a separate follow-up PR though (that I'm happy to take since I've been kinda thinking about it already and it's ReportingRound related 🙌 )